api/envoy/extensions/internal_redirect/allow_listed_routes/v3/allow_listed_routes_config.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package envoy.extensions.internal_redirect.allow_listed_routes.v3;
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.internal_redirect.allow_listed_routes.v3";
+option java_outer_classname = "AllowListedRoutesConfigProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: AllowListed routes internal redirect predicate]
```

Comment:
AllowListed -> Allow listed ?
If so I think you can remove the dictionary additions.

Reply:
Done.